code/game/objects/items/implants/implant.dm

```diff
@@ -1,12 +1,20 @@
+/**
+ * Parent class for all implants
+ */
 /obj/item/implant
 	name = "implant"
 	icon = 'icons/obj/implants.dmi'
 	icon_state = "generic" //Shows up as the action button icon
 	actions_types = list(/datum/action/item_action/hands_free/activate)
-	var/activated = TRUE //1 for implant types that can be activated, 0 for ones that are "always on" like mindshield implants
+	///true for implant types that can be activated, false for ones that are "always on" like mindshield implants
+	var/activated = TRUE
+	///the mob that's implanted with this
 	var/mob/living/imp_in = null
+	///implant color, used for visuals
 	var/implant_color = "b"
```

Comment:
Could you explain what this `b` actually represents?

Reply:
explained.